tests/src/system/basic/WskBasicTests.scala

```diff
@@ -348,6 +348,14 @@ class WskBasicTests
             assetHelper.withCleaner(wsk.rule, ruleName) {
                 (rule, name) =>
                     rule.create(name, trigger = triggerName, action = actionName)
+
+                    // to validate that the rule was created enabled, we do an update and expect CONFLICT
+                    rule.create(name, trigger = triggerName, action = actionName, update = true, expectedExitCode = CONFLICT)
```

Comment:
I'ld even add: "expecting rule update to fail with conflict, because activated rules shall not be updated"